browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -327,6 +344,11 @@ void AIChatUIPageHandler::SetChatUI(mojo::PendingRemote<mojom::ChatUI> chat_ui,
                                     SetChatUICallback callback) {
   chat_ui_.Bind(std::move(chat_ui));
   std::move(callback).Run(active_chat_tab_helper_ == nullptr);
+
+  chat_ui_->OnNewDefaultConversation(
+      active_chat_tab_helper_
+          ? std::make_optional(active_chat_tab_helper_->GetContentId())
+          : std::nullopt);
```

Comment:
Hopefully I didn't bring this up already, but I'm a little nervous that this will cause the conversation to be bound twice, via `active_chat_context.tsx`:
```
        getAPI().uiObserver.onNewDefaultConversation.addListener(() => {
          setConversationAPI(API.bindConversation(undefined))
        })
```

Reply:
Just checked - on a full page conversation `newConversation` is called once. On a sidebar `bindConversation(undefined)` is called once, so I think we're okay here :smile: